dhis-2/dhis-services/dhis-service-setting/src/main/java/org/hisp/dhis/setting/hibernate/HibernateSystemSettingStore.java

```diff
@@ -51,13 +50,6 @@ public HibernateSystemSettingStore( SessionFactory sessionFactory, JdbcTemplate
         super( sessionFactory, jdbcTemplate, publisher, SystemSetting.class, true );
     }
 
-    @Override
-    @Transactional
-    public SystemSetting getByNameTx( String name )
```

Comment:
FYI: I assume this was an odd fix for the implicit update of the audit entry in the test. Obviously reading a setting should not be a read-write transaction.